[DLStreamer] updated documentation for WSL setup (#1069)
open-edge-platform/edge-ai-libraries: libraries/dl-streamer/docs/source/get_started/install/install_guide_ubuntu_wsl2.md

Reviewer: ```suggestion
After the line with: `Defaults env_reset`, add another line with:
```

wsl --set-default Ubuntu-24.04
```

## On Linux WSL System

Open an Ubuntu WSL terminal and follow the instructions.

### Step 1: [OPTIONAL] Setup proxy

> Edit `/etc/bash.bashrc` and append lines with http_proxy and https_proxy:

Reviewer 2: Done

drwxr-xr-x 16 root root       3.5K Mar 24 16:00 ..
```

> If there is no `/dev/dri/renderD12*` directory please try running:

Reviewer 2: Done

```bash
sudo modprobe vgem
```

and check once again.

### Step 3: Add user to the `render` group

To have permission to use GPU device, the user has to belong to the `render` group.

> Please follow the instructions:

Reviewer: ```suggestion
To use a GPU device, the user has to belong to the `render` group.
Follow these steps:
```

```bash
sudo gpasswd -a ${USER} render
newgrp render
```

> Confirm that listed groups to which you belong contain the `render` group:

Reviewer: ```suggestion
Confirm that the list of groups to which you belong includes the `render` group:
```

```bash
groups ${USER}
```

### Step 4: Install drivers

> Follow the instructions to add necessary repositories and install libraries:

Reviewer: ```suggestion

```

```

### Step 5: Add OpenVINO™ Toolkit and Deep Learning Streamer repositories

Follow the instructions to add necessary repositories for Deep Learning Streamer Framework and OpenVINO™ Toolkit:

```bash
cd $HOME
sudo -E wget -O- https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB | gpg --dearmor | sudo tee /usr/share/keyrings/intel-gpg-archive-keyring.gpg > /dev/null
sudo -E wget -O- https://apt.repos.intel.com/edgeai/dlstreamer/GPG-PUB-KEY-INTEL-DLS.gpg | sudo tee /usr/share/keyrings/dls-archive-keyring.gpg > /dev/null
echo "deb [signed-by=/usr/share/keyrings/dls-archive-keyring.gpg] https://apt.repos.intel.com/edgeai/dlstreamer/ubuntu24 ubuntu24 main" | sudo tee /etc/apt/sources.list.d/intel-dlstreamer.list
sudo bash -c 'echo "deb [signed-by=/usr/share/keyrings/intel-gpg-archive-keyring.gpg] https://apt.repos.intel.com/openvino/2025 ubuntu24 main" | sudo tee /etc/apt/sources.list.d/intel-openvino-2025.list'
```

### Step 6: Install Deep Learning Streamer Pipeline Framework

Follow the instructions to install Deep Learning Streamer and its dependencies:

```bash
sudo apt update
sudo apt install intel-dlstreamer
```

### Step 7: Download yolo11s model

> If you would like to execute sample pipelines, please download yolo11s model as the sample one for these pipelines:

Reviewer 2: Done

sudo apt-get install intel-dlstreamer
```

### [Optional] Step 6: Do optional steps.

> All optional steps including download models and run sample application could be run as described in [Install Guide Ubuntu](./install_guide_ubuntu.md)

Reviewer 3: A minor style fix
'All optional steps, including downloading models and running the sample application, can be completed by following the instructions in the [Install Guide Ubuntu].'
